pyproject.toml

```diff
@@ -58,4 +52,44 @@ messages_control.disable = [
   "invalid-name",
   "protected-access",
   "missing-module-docstring",
+  "unused-argument",  # covered by Ruff ARG
+]
+
+[tool.ruff]
+select = [
```

Comment:
Newest version of ruff has [PEP8-naming rules implemented](https://beta.ruff.rs/docs/rules/#pep8-naming-n). We should enable those too

Reply:
```
pybind11/setup_helpers.py:274:7: N801 Class name `build_ext` should use CapWords convention
tests/test_exceptions.py:292:7: N818 Exception name `FlakyException` should be named with an Error suffix
tests/test_pytypes.py:18:5: N802 Function name `test_handle_from_move_only_type_with_operator_PyObject` should be lowercase
tests/test_sequences_and_iterators.py:157:11: N818 Exception name `BadLen` should be named with an Error suffix
Found 4 errors.
```

Not sure it really matters in tests, and the one in the project is wrong. Guess I can just use `--add-noqa` and inject these four skips, though.

Reply:
Okay, skipped naming rules for tests, and noqa'd the only non-test one.